backend/tracim_backend/views/search_api/schemas.py

```diff
@@ -0,0 +1,245 @@
+        validate=regex_string_as_list_of_string,
+        description="select contents in these workspaces",
+    )
+    author__public_names = StrippedString(
+        required=False,
+        validate=regex_string_as_list_of_string,
+        description="select contents by these authors",
+    )
+    last_modifier__public_names = StrippedString(
+        required=False,
+        validate=regex_string_as_list_of_string,
+        description="select contents by these authors",
+    )
+    file_extensions = StrippedString(
+        required=False,
+        validate=regex_string_as_list_of_string,
+        description="select contents with these file extensions",
+    )
+    statuses = StrippedString(
+        required=False,
+        validate=regex_string_as_list_of_string,
+        description="select contents with these statuses",
+    )
+    created_from = marshmallow.fields.DateTime(required=False, format=DATETIME_FORMAT)
+    created_to = marshmallow.fields.DateTime(required=False, format=DATETIME_FORMAT)
+    updated_from = marshmallow.fields.DateTime(required=False, format=DATETIME_FORMAT)
+    updated_to = marshmallow.fields.DateTime(required=False, format=DATETIME_FORMAT)
+
+    # @post_load
+    # def make_advanced_search_content_filter(self, data: typing.Dict[str, typing.Any]) -> object:
+    # return AdvancedContentSearchParameters(
+    # workspace_names=data["workspace_names"],
+    # author__public_names=data["author__public_names"],
+    # last_modifier__public_names=data["last_modifier__public_names"],
+    # file_extensions=data["file_extensions"],
+    # search_fields=data["search_fields"],
+    # statuses=data["statuses"],
+    # created_from=data["created_from"],
+    # created_to=data["created_to"],
+    # updated_from=data["updated_from"],
+    # updated_to=data["updated_to"],
+    # )
+
+
+class WorkspaceSearchSchema(marshmallow.Schema):
+    workspace_id = marshmallow.fields.Int(example=4, validate=strictly_positive_int_validator)
+    slug = StrippedString(example="intranet")
+    label = StrippedString(example="Intranet")
+
+
+class ContentSearchSchema(ContentDigestSchema, UserInfoContentAbstractSchema):
+    score = marshmallow.fields.Float()
+    workspace = marshmallow.fields.Nested(WorkspaceSearchSchema)
+    path = marshmallow.fields.List(marshmallow.fields.Nested(ContentMinimalSchema))
+    is_active = marshmallow.fields.Boolean()
+    comments_count = marshmallow.fields.Integer(example=12, validate=positive_int_validator)
+    content_size = marshmallow.fields.Integer(
+        example=1200, description="Content size in bytes", validate=positive_int_validator
+    )
+
+
+class ContentSearchResultSchema(marshmallow.Schema):
+    contents = marshmallow.fields.Nested(ContentSearchSchema, many=True)
+    total_hits = marshmallow.fields.Integer()
+    is_total_hits_accurate = marshmallow.fields.Boolean()
+
+
+class FacetCountSchema(marshmallow.Schema):
+    key = marshmallow.fields.String("The value of this field")
```

Comment:
According to the issue `key` should be `value`. I do not especially care about the naming, but update the issue accordingly and ask @gbonaspetti if she already has implemented the frontend part for this.